app/src/main/java/io/gnosis/safe/ErrorHandler.kt

```diff
@@ -53,6 +54,7 @@ sealed class Error(
 
     object Error104 : Error(104, 104, R.string.error_network_unknown_host_reason, R.string.error_network_unknown_host_fix)
     object Error400 : Error(400, 400, R.string.error_network_request_reason, R.string.error_network_request_fix)
+    object ErrorCloudMessagingTokenIsLinkedToAnotherDevice : Error(400, 400, R.string.error_network_request_reason, R.string.error_network_request_fix)
```

Comment:
could you please define error in https://docs.google.com/document/d/1foQ9DSMgO2iN-6u4JsJIDJWB7mUHX-D8ApZWvx6oKAo/edit?ts=5ff5e055# with separate id and use our naming scheme (Error<Id>) . Please check with ios if this should be an android error (17xx) or if ios is also affected. Then it would be common client error (11xx)

Reply:
I hesitated to add this because we're never going to show it to the user. Why have it in the list, if it is only internally used?

Reply:
I think having it in spec would improve overview about errors that we handle on the client